app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,8 @@
+.footer {
+  text-align: center;
+
```

Comment:
Think would be good to add margin on the footer and navbar so that it does not interfere with the body of the page